File: EJERCICIOS EN CLASE/ProgramaPromedio/promedio.java
import java.util.Scanner;
public class promedio {
    public static void main(String[] args) {
        float nt1, nt2, nt3, promedio;
        try (var sc = new Scanner(System.in)) {
            System.out.println("Ingrese la nota 1: ");

        nt1 = sc.nextFloat();

        System.out.println("Ingrese la nota 2: ");

        nt2 = sc.nextFloat();

        System.out.println("Ingrese la nota 3: ");

        nt3 = sc.nextFloat();

        promedio = (nt1 + nt2 + nt3) / 3;
        
        System.out.println("Su primera nota es: " + nt1);
        System.out.println("Su segunda nota es: " + nt2);
        System.out.println("Su tercera nota es: " + nt3);
        System.out.println("Su promedio es: " + promedio);
    }
    }
}

File: EJERCICIOS EN CLASE/ProgramaPromedio/promedio.cpp
#include <iostream>
using namespace std;
int main(){
    float nt1, nt2, nt3, promedio;

    cout << "Ingrese la nota 1: ";
    cin >> nt1;

    cout << "Ingrese la nota 2: ";
    cin >> nt2;

    cout << "Ingrese la nota 3: ";
    cin >> nt3;

    promedio = (nt1 + nt2 + nt3) / 3;
    cout << "Su promedio es: " << promedio << endl;
    return 0;
}